coverity-scan.sh

```diff
@@ -44,4 +44,4 @@ curl --progress-bar --form token="${token}" \
   --form file=@netdata-coverity-analysis.tgz \
   --form version="${version}" \
   --form description="netdata, real-time performance monitoring, done right." \
-  https://scan.coverity.com/builds?project=firehol%2Fnetdata
+  https://scan.coverity.com/builds?project=netdata%2Fnetdata
```

Comment:
Not active yet

Reply:
added it to the list above

Reply:
It is already on the list here: https://github.com/netdata/netdata/issues/4190#issuecomment-422779381 :)

Reply:
Let's not duplicate lists